pages/client_page.vue

```diff
@@ -0,0 +1,240 @@
+}
+
+button {
+    grid-column: 1;
+    grid-row: 1;
+    justify-self: start;
+    font-size: 18px;
+    color: $text500;
+    align-self: end;
+    padding: 10px 20px;
+}
+
+div.client_profile {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: $background500;
+    grid-row: 1;
+    grid-column: 1 / span 2;
+    justify-self: end;
+    align-self: end;
+}
+
+.client_profile > div {
+    margin-right: 20px;
+    font-size: 30px;
+}
+
+.client_profile > img {
+    width: 100px;
+    height: 100px;
+    border-radius: 100%;
+}
+
+.client_profile > img, div {
+    float: left;
+}
+
+.client_data {
+    grid-column: 1;
+    grid-row: 2;
+    font-size: 30px;
+    padding: 20px 40px;
+    display: grid;
+    grid-template-columns: 340px 500px;
+    column-gap: 20px;
+    justify-items: stretch;
+}
+
+.data_header {
+    margin-bottom: 30px;
+}
+
+.username, .names, .email {
+    grid-column: 1;
+    font-size: 24px;
+}
+
+.label_username, .label_names, .label_email {
+    grid-column: 1;
+    font-size: 15px;
+}
+
+.subscriptions {
+    border: 2px $text500 solid;
+    border-radius: 10px;
+    grid-column: 2;
+    grid-row: 1 / span 7;
+    justify-self: stretch;
+    align-self: stretch;
+    row-gap: 10px;
+    padding: 0 0 20px 0;
+}
+
+.subscriptions_header {
+    font-size: 26px;
+    text-align: center;
+    padding: 20px 0;
+    background: $primary500;
+    border-top-right-radius: 10px;
+    border-top-left-radius: 10px;
+    width: 501px;
+    transform: translate(-2px, -2px);
+}
+
+.balance_state {
+    font-size: 20px;
+    padding: 20px 20px;
+    width: 460px;
+}
+
+.bundles_label {
+    font-size: 14px;
+    padding: 10px 20px;
+    width: 460px;
+}
+
+.bundles {
+    padding: 0 20px;
+    width: 460px;
+    margin-top: 10px;
+}
+
+.bundle {
+    background-color: #000000;
+    width: 460px;
+    height: 50px;
+    margin-bottom: 10px;
+    border-radius: 10px;
+    text-align: center;
+}
+
+.buttons {
+    grid-column: 1;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    column-gap: 20px;
+    background-color: $background500 !important;
+    padding: 0 !important;
+}
+
+.buttons > div {
+    background-color: $background400;
+    font-size: 22px;
+    border-radius: 10px;
+}
+
+.client_manage_button {
+    grid-column: 1;
+}
+
+.client_edit_button {
+    grid-column: 2;
+}
+
+.client_manage_button, .client_edit_button {
+    display: grid;
+    align-items: center;
+    justify-items: center;
+}
+
+.client_card {
+    grid-column: 2;
+    grid-row: 2 / span 2;
+    font-size: 30px;
+    padding: 20px 40px;
+}
+
+.card_header {
+    margin-bottom: 40px;
+}
+
+.card_number, .card_valid_date, .card_type {
+    grid-column: 1;
+    font-size: 24px;
+    width: 360px;
+    margin-bottom: 10px;
+}
+
+.label_card_number, .label_card_valid_date, .label_card_type {
+    grid-column: 1;
+    font-size: 15px;
+    width: 360px;
+    margin-bottom: 40px;
+}
+
+.label_card_type {
```

Comment:
jestem pewien, że można to bardziej uprościć:

![image](https://github.com/documentoviscode/splashy-frontend/assets/81694867/eb975d5a-10be-4c22-902a-aea6b6708634) 
w tym miejscu 3 razy jest taki sam styl, ale klasy - lepiej byłoby zrobić div dla każdej sekcji, a w nim dwa elementy tekstowe - teraz style są dość niezrozumiałe na pierwszy rzut oka